### DISCUSSION TITLE
New Btcpayserver update 1.7.3

### DISCUSSION THREAD

**Discussion by Fenix2013** [btcpayserver/btcpayserver#4479] [General]
I'm currently using Btcpay Server version 1.7.1 and I can't update it to a newer version (neither 1.7.2 nor 1.7.3). I tried it in the user interface and also in the Putty virtual machine, but unfortunately it doesn't work). Could you advise me how to solve it.

Thank you in advance and I look forward to your reply as soon as possible.

Kind regards

**Comment by dennisreimann** [comment]
What is you deployment method? Are you using the Docker deployment, a web hoster, something else?
Did upgrades work before and how did you upgrade then?
What error messages do you get?

**Reply by Fenix2013** [reply]
I'm using a web hoster (I'm using Luna Node).I have been using it for more than 2 years and all that time I updated without problems, I never had the slightest problem.I've always updated in user interface (Server Settings-Maintenance-Update) and I never have any problem.The last time I tried to update it they also use SSH login in Putty virtual machine but equally unsuccessful (a message appeared to me: fatal: unable to access 'https://github.com/btcpayserver/btcpayserver-docker/': Could not resolve host: github.com").

**Reply by dennisreimann** [reply]
Ok, does this still happen? It looks like a temporary connection problem with GitHub. If it persists, can you reach other services like `ping google.com`?

**Reply by Fenix2013** [reply]
Yes,it still happens.I haven't tried reach other services like ping google com yet (to be honest, I didn't think of it).

**Reply by Fenix2013** [reply]
Dear Dennis,
finally I solved my problem with the help of Luna Node support team and my Btcpay Server is updated to the latest version 1.7.3.
But I also want to express my gratitude to you for your willingness and assistance in solving my problem.

Great regards

**Reply by Fenix2013** [reply]
![Screenshot_20221229_043720_Proton Mail.jpg](https://user-images.githubusercontent.com/57800986/209900165-a7d8cffa-10bf-459b-b288-f48f76b64aac.jpg)

![Screenshot_20221229_043108_Proton Mail.jpg](https://user-images.githubusercontent.com/57800986/209900180-52f0787e-c688-4d40-bd5f-3ade2070af83.jpg)

![Screenshot_20221229_041847_Samsung Internet.jpg](https://user-images.githubusercontent.com/57800986/209900190-454bd0ca-d3d8-4d5e-8a86-d4288b36313b.jpg)

**Reply by dennisreimann** [reply]
Glad it's working for you now! 😀
